frontend/src/components/common/header/navbar.jsx

```diff
@@ -78,12 +78,12 @@ class NavBar extends Component {
                     id="nav-dropdown"
                   >
                     <NavDropdown.Item>
-                      <HashLink eventKey="4.1" to="/resources#hospital">
+                      <HashLink eventKey="4.1" to="/resources#hospitals">
                         Hospitals
                       </HashLink>
                     </NavDropdown.Item>
                     <NavDropdown.Item>
-                      <HashLink eventKey="4.2" to="/resources#uc">
+                      <HashLink eventKey="4.2" to="/resources#urgent_care">
```

Comment:
most of these links are not working as we need to update this to the correct id in the html. So this one should be `urgent-care`

the shelter one needs to be `shelters`

the emergency resources one should be `emergency-resources` 

and i think these links should not be blue?

Reply:
Routing was fixed. I changed the color for the other dropdown links, the Resources one remains yellow and does not appear to have an :active property, the other ones look fine. I'm thinking this is  because we should `import { HashLink  as Navlink}...` ---> this could be a whole new ticket, dunno.